packages/server/src/models/items/storage/StorageDriverS3.ts

```diff
@@ -62,6 +63,7 @@ export default class StorageDriverS3 extends StorageDriverBase {
 
 			return stream2buffer(response.Body);
 		} catch (error) {
+			if (error?.Code === 'InternalError') throw new ErrorBadGateway('Storage driver server returned an internal error. Try again later.');
```

Comment:
Bad Gateway makes sense but I feel it might be a bit too transparent since it reveals how it's working internally and in particular that the S3 driver is being used. Instead maybe some more generic like Service Unavailable would be better

Reply:
Do you think the error message should be be changed too?